shadows/framework/src/main/java/org/robolectric/shadows/ShadowParcelFileDescriptor.java

```diff
@@ -163,4 +216,46 @@ protected void close() throws IOException {
     @Direct
     void close();
   }
+
+  /**
+   * Tracks all the opened files opened by {@link ShadowParcelFileDescriptor#open(File, int)}.
+   *
+   * <p>Only for internal used by PFD.
+   *
+   * <p>The concept of "OpenedFileTable" is quite similar to *nix.
+   *
+   * <p>Note that is not supposed to be run on Windows, for the key `fd` is only available in *nix.
+   * In Windows the `handle` should be used instead of the `fd`.
+   */
+  static class OpenedFileTable {
```

Comment:
private static class?

Reply:
Done. `OpenedFileTable` is only for internal used. The visibility is `private` now.